lib/Scheduler/sched_region.cpp

```diff
@@ -414,13 +427,16 @@ FUNC_RESULT SchedRegion::FindOptimalSchedule(
       bestSched = bestSched_ = lstSched;
       bestSchedLngth_ = heuristicScheduleLength;
       bestCost_ = hurstcCost_;
+      BestSpillCost_ = HurstcSpillCost_;
     }
     // B) Heuristic was never run. In that case, just use ACO and run with its
     // results, into B&B.
     else if (!HeuristicSchedulerEnabled) {
       bestSched = bestSched_ = AcoSchedule;
       bestSchedLngth_ = AcoScheduleLength_;
       bestCost_ = AcoScheduleCost_;
+      // need to set spillCost
```

Comment:
Is this a TODO comment? Was this addressed?

Reply:
Yes this was meant to be a TODO -- good catch on this. Thank you